app/src/types/paymentDetails.ts

```diff
@@ -0,0 +1,23 @@
+import { ObjectOf } from './generic';
+
+export type PaymentDetail = {
```

Comment:
I think this type can be in types/payment.ts

types folder is organized by resources